config/routes.rb

```diff
@@ -4,7 +4,12 @@
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
-
+  resources :users do
+    resources :posts do
+      resources :comments
+      resource :likes
+    end
+  end
```

Comment:
- [ ] We don't need the comments and likes routes for now, so kindly remove them.

- [ ] We don't want to expose all the available routes for users and posts yet, we only need routes for `index` and `show` actions in the respective controllers.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```